### PR TITLE
feat: integrate with API app and add BrandProxy

### PR DESCRIPTION
**Pull request by GarfDev**
**What**
- Integrate with API app
- Add BrandProxy
- Remove Skin Model/Service

### CODE REVIEW

File: fixture/db/client.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
             "client_secret_expires_at": 0,
             "client_secret": "VVrUT7n82KNcuh947RAI-dqY79K8r47k_Xk9WvdKXl_viltV5kzX5oNDjen97vr0pBRSlv3oGZvz3_ka1136LQ",
             "redirect_uris": ["https://localhost:8082/signin-oidc"],
-            "scope": "openid pools:read pools:write withdrawals:read rewards:write promotions:read promotions:write deposits:write deposits:read"
+            "scope": "openid rewards:read erc20:read erc721:read withdrawals:read withdrawals:write deposits:read deposits:write account:read account:write memberships:read memberships:write promotions:read relay:write"
```
Comment (peterpolman) [inline fixture/db/client.json:22]: @GarfDev Could you explain the changes you made here? It seems not ok to me that pool permissions are removed and a relay permission is added.

Comment (GarfDev) [reply fixture/db/client.json:22]: I have copied changes from oicd.ts of dashboard into here

Comment (peterpolman) [reply fixture/db/client.json]: Looking at the specific scopes I think you have mistakenly copied the scopes for the wallet in here! Please verify that:)

**Review by peterpolman — APPROVED** [review]

File: fixture/db/client.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
             "client_secret_expires_at": 0,
             "client_secret": "VVrUT7n82KNcuh947RAI-dqY79K8r47k_Xk9WvdKXl_viltV5kzX5oNDjen97vr0pBRSlv3oGZvz3_ka1136LQ",
             "redirect_uris": ["https://localhost:8082/signin-oidc"],
-            "scope": "openid pools:read pools:write withdrawals:read rewards:write promotions:read promotions:write deposits:write deposits:read"
+            "scope": "openid account:read pools:read pools:write erc20:read erc20:write erc721:read erc721:write rewards:read rewards:write withdrawals:read deposits:read deposits:write promotions:read promotions:write widgets:write widgets:read transactions:read payments:read payments:write"
```
Comment (peterpolman) [inline fixture/db/client.json:22]: @GarfDev  Shouldn't the `brands:read` and `brands:write`scopes be whitelisted for the dashboard? Thats is where the resource is managed right?

I noticed they are added in the oidc request config over here:
https://github.com/thxprotocol/dashboard/pull/60/files

**Pull request merged**